geonotebook/layers.py

```diff
@@ -21,25 +105,35 @@ class GeonotebookLayer(object):
     # some kind of functionality (e.g. the annotatoin layer).
     _expose_as = None
 
-    _type = None
-
-    def __init__(self, name, remote, **kwargs):
+    def __init__(self, name, remote, data, **kwargs):
```

Comment:
to me, data or vis_url are data. what are all the options users are expected to provide for kwargs?